### PR TITLE
feat: deployment

### PR DESCRIPTION
**Pull request by marybrown123**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,14 +1,16 @@
 version: '3.8'
```
Comment (wojdanowski) [inline docker-compose.yml:1]: please make sure that volumes for db are correctly set up. Try to add something to the database via API and make sure that data is persisted after re-deploing/restarting the container

File: .circleci/config.yml [hunk 2]
```diff
@@ -82,7 +110,23 @@ jobs:
     steps:
       - checkout
       - greetings-from-branch
-  
+  deploy:
```
Comment (wojdanowski) [inline .circleci/config.yml:113]: Deployment should be triggered only after you marge changes to the master branch.
You do not want to deploy changes from other branches

Comment (marybrown123) [reply .circleci/config.yml:113]: I know, but i made it like this only to test it, i will add option for triggering only after merging to master :)

File: docker-compose.yml (hunk 1 above)
Comment (marybrown123) [reply docker-compose.yml:1]: I checked it and data is persisted

**Review by wojdanowski — APPROVED** [review]

**Pull request merged**